src/include/test/unit_utils.cc

```diff
@@ -33,6 +33,14 @@
 #include "config.h"
 #include "utils/utils.h"
 
+namespace {
```

Comment:
Ah -- interesting -- I see how this is useful now.

Reply:
Yea, but it still isn't good :/, since this operator is defined in header it pollutes the top level namespace. It works, I just don't like it, I would prefer if it was only defined in compilation unit.